src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,177 @@
+                () -> registrationService.register(testUser));
+    }
+
+    @Test
+    void register_passwordGreaterThanMinimum_ok() throws RegistrationException {
+        String password = CORRECT_PASSWORD;
+        testUser.setPassword(password);
+        registrationService.register(testUser);
+        Assertions.assertEquals(password,testUser.getPassword());
+    }
+
+    @Test
+    void register_passwordLesserThanMinimum_notOk() {
+        testUser.setPassword(INCORRECT_PASSWORD);
+        Assertions.assertThrows(RegistrationException.class,
+                () -> registrationService.register(testUser));
+    }
+
+    @Test
+    void register_emptyPassword_notOk() {
+        testUser.setPassword(EMPTY_LINE);
+        Assertions.assertThrows(RegistrationException.class,
+                () -> registrationService.register(testUser));
+    }
+
+    @Test
+    void register_minimumPassword_ok() throws RegistrationException {
+        String password = BORDER_PASSWORD;
+        testUser.setPassword(password);
+        registrationService.register(testUser);
+        Assertions.assertEquals(password,testUser.getPassword());
+    }
+
+    @Test
+    void register_nullPassword_notOk() {
+        testUser.setPassword(null);
+        Assertions.assertThrows(RegistrationException.class,
+                () -> registrationService.register(testUser));
+    }
+
+    @Test
+    void register_nullAge_notOk() {
+        testUser.setAge(0);
+        Assertions.assertThrows(RegistrationException.class,
+                () -> registrationService.register(testUser));
+    }
+
+    @Test
+    void register_negativeAge_notOk() {
+        testUser.setAge(NEG_AGE);
+        Assertions.assertThrows(RegistrationException.class,
+                () -> registrationService.register(testUser));
+    }
+
+    @Test
+    void register_borderAge_ok() throws RegistrationException {
+        int age = BORDER_AGE;
+        testUser.setAge(age);
+        registrationService.register(testUser);
+        Assertions.assertEquals(age,testUser.getAge());
+    }
+
+    @Test
+    void register_allowableAge_ok() throws RegistrationException {
+        int age = CORRECT_AGE;
+        testUser.setAge(age);
+        registrationService.register(testUser);
+        Assertions.assertEquals(age,testUser.getAge());
+    }
+
+    @Test
+    void register_unlawfulAge_notOk() {
+        testUser.setAge(IN_CORRECT_AGE);
+        Assertions.assertThrows(RegistrationException.class,
+                () -> registrationService.register(testUser));
+    }
+
+    @Test
+    void register_allowableUser_ok() throws RegistrationException {
+        Assertions.assertEquals(testUser,registrationService.register(testUser));
+    }
+
+    @Test
+    void register_registrationNewUser_ok() throws RegistrationException {
+        Storage.people.add(testUser);
+        User user = registrationService.register(testUserNew);
+        Assertions.assertEquals(2,Storage.people.size());
```

Comment:
```suggestion
        Assertions.assertEquals(2, Storage.people.size());
```
autoformat this file (Ctrl Alt L), a comma should always be followed by a space